media_commons_booking_app/src/client/routes/booking/bookingProvider.tsx

```diff
@@ -80,9 +83,15 @@ export function BookingProvider({ children }) {
     fetchIsSafetyTrained();
   }, [fetchIsSafetyTrained]);
 
-  useEffect(() => {
-    setIsStudent(role === Role.STUDENT);
-  }, [role]);
+  const checkSafetyTraining = (roomIds = []) => {
```

Comment:
this shouldn't be a function in the provider, but rather the provider should have a variable stating whether the given user is subject to safety approval. I think anything related to the roomIds should stay out of the provider

Reply:
instead the provider could expose `setNeedsSafetyTraining`

Reply:
I agree with you! I updated!